src/ui/common/src/components/pages/artifact/id/index.tsx

```diff
@@ -0,0 +1,207 @@
+import { faCircleCheck } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { CircularProgress } from '@mui/material';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import {
+  ArtifactResult,
+  handleGetArtifactResults,
+} from '../../../../reducers/workflow';
+import { AppDispatch, RootState } from '../../../../stores/store';
+import UserProfile from '../../../../utils/auth';
+import { Data, DataSchema } from '../../../../utils/data';
+import { exportCsv } from '../../../../utils/preview';
+import DefaultLayout from '../../../layouts/default';
+import { Button } from '../../../primitives/Button.styles';
+import KeyValueTable, {
+  KeyValueTableType,
+} from '../../../tables/KeyValueTable';
+import StickyHeaderTable from '../../../tables/StickyHeaderTable';
+import { LayoutProps } from '../../types';
+
+const kvSchema: DataSchema = {
+  fields: [
+    { name: 'Title', type: 'varchar' },
+    { name: 'Value', type: 'varchar' },
+  ],
+  pandas_version: '0.0.1', // TODO: Figure out what to set this value to.
+};
+
+const mockMetrics: Data = {
+  schema: kvSchema,
+  data: [
+    ['avg_churn', '0.04'],
+    ['avg_workflows', '455'],
+    ['avg_users', '1.2'],
+    ['avg_users', '5'],
+  ],
+};
+
+const mockChecks: Data = {
+  schema: kvSchema,
+  data: [
+    ['reasonable_churn', 'True'],
+    ['wf_count_small', 'False'],
+    ['bounds_check', 'True'],
+    ['avg_users_check', 'False'],
+    ['warning_check', 'Warning'],
+    ['none_check', 'None'],
+  ],
+};
+
+type ArtifactDetailsHeaderProps = {
+  artifactName: string;
+  createdAt?: string;
+  sourceLocation?: string;
+};
+
+const ArtifactDetailsHeader: React.FC<ArtifactDetailsHeaderProps> = ({
+  artifactName,
+  // TODO: add these back once we have support for getting createdAt and sourceLocation.
```

Comment:
@likawind Are we able to get this information yet with the latest changes that you've merged recently? We'll also need similar info for operators too if possible.